pkg/kubelet/kuberuntime/security_context_others.go

```diff
@@ -0,0 +1,51 @@
+// +build !windows
+
+/*
```

Comment:
the pattern in file naming is `security_context_windows.go` and `security_context.go` In couple places I saw `_unix` or `_linux` suffices. I'd vote for consistency and drop `_other`

Reply:
Here I have a very uncertain thing is that other platforms besides Linux and Windows support this feature, so I chose to use others~~

Reply:
thank you for pointing out. Somehow all examples I saw was not using this pattern. I guess it's ok. If not previous examples my suggestion would be `security_context_non_windows.go`. It may be the most descriptive name

Reply:
About this, I went to look at some of the existing designs of kubernetes.

- If the platform support is very clear, it is generally `*_linux.go`, `*_windows.go`, `*_unspport.go`. 
- If it is possible to distinguish between windows and others, it is generally `*_others.go`, `*_windows.go`.

I think it’s better to follow the existing naming rules, otherwise it will confuse people , what do you think?

Reply:
sure, lets keep it. It looks consistent with the pattern. Thank you for looking into it